CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 - Fix a false positive for `FactoryBot/ConsistentParenthesesStyle` when using traits and omitting hash values. ([@thejonroberts])
 - Make RuboCop FactoryBot work as a RuboCop plugin. ([@bquorning])
+- Fix a false negative for `FactoryBot/SyntaxMethods` when method is used inside a module. ([@lovro-bikic])
```

Comment:
Please move above, to unreleased

Reply:
Fixed, thank you. The rebase somehow pushed it from unreleased down to a released version (I had this branch sitting for a few months) 🤔